templates/github/.ci/ansible/settings.py.j2.copy

```diff
@@ -22,23 +22,29 @@ API_ROOT = {{ api_root | repr }}
 
 {% if pulp_scenario_settings is defined and pulp_scenario_settings %}
 {% for key, value in pulp_scenario_settings.items() %}
+{% if not (s3_test | default(false) and key == "STORAGES") %}
 {{ key | upper }} = {{ value | repr }}
+{% endif %}
 {% endfor %}
 {% endif %}
 
 {% if s3_test | default(false) %}
+MEDIA_ROOT: ""
+S3_USE_SIGV4 = True
+{% if "STORAGES" in pulp_scenario_settings %}
+STORAGES = {{ pulp_scenario_settings["STORAGES"] }}
```

Comment:
Is the `| repr` the reason why you need to single out this line? Is this related to the `"{{ minio_access_key }}"` part?

Reply:
TBH I dont know, but maybe.
The reason was to keep the alternatives explicit